### ISSUE TITLE
NestJs resolvers not working when GraphQL schema property is used

### ISSUE THREAD

**Issue by juicycleff** [nestjs/graphql#288]
NestJs resolvers not working when schema is property used. The resolvers don't work.

```typescript
@Injectable()
export class GraphqlOptions implements GqlOptionsFactory {
  createGqlOptions(): Promise<GqlModuleOptions> | GqlModuleOptions {
    return {
      path: '/graphql',
      schema:  buildFederatedSchema([
        typeDefsHubType, typeDefsHubs,
      ]),
      installSubscriptionHandlers: true,
      resolverValidationOptions: {
        requireResolversForResolveType: false,
      },
      playground: {
        settings: {
          'editor.theme': 'light',
        },
      },
    };
  }
}
```

**Comment by marcus-sa** [comment]
I'm trying to utilize `@apollo/federation` as well, but without any luck so far.

**Comment by marcus-sa** [comment]
Check this out https://github.com/marcus-sa/nest-graphql-microservices-example

**Comment by kamilmysliwiec** [comment]
Fixed in 6.3.1 :)
